Mathlib/MeasureTheory/Integral/Bochner.lean

```diff
@@ -1271,6 +1271,20 @@ theorem integral_eq_zero_iff_of_nonneg_ae {f : α → ℝ} (hf : 0 ≤ᵐ[μ] f)
     simp only [Pi.zero_apply, ofReal_eq_zero]
   · exact (ENNReal.measurable_ofReal.comp_aemeasurable hfi.1.aemeasurable)
 
+lemma ae_eq_of_ae_le_of_integral_eq {f g : α → ℝ} (hf : Integrable f μ)
+    (hg : Integrable g μ) (h_le : f ≤ᵐ[μ] g) (h_eq : ∫ a, f a ∂μ = ∫ a, g a ∂μ) :
+    f =ᵐ[μ] g := by
```

Comment:
Can you make this an `iff` (with `h_eq`)?

Reply:
Sorry, I did not see that second comment. I'll add those lemmas.